src/test/smart-components/myUserAccess/MUAContent.test.js

```diff
@@ -45,6 +45,6 @@ describe('<MUAContent />', () => {
       );
     });
 
-    expect(wrapper.find(Title).last().text()).toEqual('Your permissions');
+    expect(wrapper.find(Title).last().text()).toEqual('Red Hat Insights permissions');
```

Comment:
@ryelo Should the bundle name still contain the "Red Hat" part?

Reply:
We can remove it here if it's not too much of a hassle